JavaScript/Active_pull_request_commentors/script.js

```diff
@@ -0,0 +1,78 @@
+const fetch = require("node-fetch");
+const createCsvWriter = require("csv-writer").createObjectCsvWriter;
+
+let OWNER = `HarshCasper`; // replace with repository's owner username
+let REPO_NAME = `Rotten-Scripts`; // replace with repositort's name
+let NO_OF_REQUEST = 10; // replace with no. of latest Active pull request data you want to access
+let obj;
+const accessToken = ""; // replace with your own github access token
```

Comment:
Keep a `.env` file where the access token can be kept. Push a `.env.example` in the PR to help the users know how to fill the access token

Reply:
Do I have to provide the steps to generate personal github access token in .env.example itself?

Reply:
It would be preferred,